Controller/RegistrationFOSUser1Controller.php

```diff
@@ -59,16 +58,25 @@ public function registerAction()
                 $route = $this->get('session')->get('sonata_basket_delivery_redirect');
 
                 if (null !== $route) {
+                    // NEXT_MAJOR: remove the if block
+                    $deprecationMessage = <<<'EOT'
```

Comment:
Not sure it's a vim problem after all. I don't know how to format it, let's ask styleCI.